src/signals/incident/components/form/MapSelectors/Asset/Selector/SelectionPanel/SelectionPanel.tsx

```diff
@@ -131,17 +131,13 @@ const SelectionPanel: FC<SelectionPanelProps> = ({
             label={language.unregistered || 'Het object staat niet op de kaart'}
           />
 
-          {showObjectIdInput && (
+          {showObjectIdInput && language.unregisteredId && (
             <>
               <Label
                 htmlFor="unregisteredAssetInput"
                 label={
                   <>
-                    <strong>
-                      {language.unregisteredId ||
-                        'Wat is het nummer van het object?'}
-                    </strong>{' '}
-                    (niet verplicht)
+                    <strong>{language.unregisteredId}</strong> (niet verplicht)
```

Comment:
Why is `unregisteredId` required? Its value is `undefined` in `src/signals/incident/definitions/wizard-step-2-vulaan/openbaarGroenEnWater.ts`

Reply:
Because trees don't have a number, and then you don't want to show the field for typing a tree number